frontend/packages/shared/src/version-control/CloneModal.tsx

```diff
@@ -19,20 +19,17 @@ export interface ICloneModalProps {
 
 export function CloneModal(props: ICloneModalProps) {
   const [hasDataModel, setHasDataModel] = useState(false);
-
-  const copyGitUrl = () => {
-    const textField = document.querySelector('#repository-url');
-    (textField as any).select();
-    document.execCommand('copy');
-  };
+  const { org, app } = useParams();
+  const gitUrl = window.location.origin.toString() + repositoryGitPath(org, app);
+  const copyGitUrl = () => navigator.clipboard.writeText(gitUrl);
 
   const canCopy = () => {
```

Comment:
This is a small detail so I don't expect you to do anything about it. But here we could avoid a function call by simply referencing the value in a variable instead. This code was here before your commit, so you decide if you do something with it or not, 
```suggestion
  const canCopy = document.queryCommandSupported ?  document.queryCommandSupported('copy') : false;
```